src/internal/operations/backup.go

```diff
@@ -516,6 +516,38 @@ func matchesReason(reasons []kopia.Reason, p path.Path) bool {
 	return false
 }
 
+// getNewPathRefs returns
+//  1. the new RepoRef for the item if it needs merged
+//  2. the new locationPath
+//  3. if the location was likely updated
+//  4. any errors encountered
+func getNewPathRefs(
+	dataFromBackup kopia.DetailsMergeInfoer,
+	entry *details.DetailsEntry,
+	repoRefPB *path.Builder,
```

Comment:
Nit: not needed, and makes the name less readable
```suggestion
	repoRef *path.Builder,
```